doc/development_guide/testing.rst

```diff
@@ -72,6 +72,15 @@ You can also use ``# +n: [`` with n an integer if the above syntax would make th
 
 If you need special control over Pylint's configuration, you can also create a .rc file, which
 can have sections of Pylint's configuration.
+The .rc file can also contain a section ``[testoptions]`` to pass options for the functional
+test runner. The following options are currently supported:
+
+    "min_pyver": Minimal python version required to run the test
+    "max_pyver": Maximum python version required to run the test
```

Comment:
This isn't completely true. Whereas `min_pyver` acts inclusively, `max_pyver` is exclusive due to the way tuple comparisons and `sys.version_info` work in Python.

I.e. an alpha release would be `(3, 11, 0, 'alpha', 0) > (3, 11)`. Thus setting  `max_pyver = 3.11` effectively excludes `3.11` and above.

IMO that is the desired behavior as otherwise one would need to say something like `3.10.99` to  exclude `3.11+`. So just the documentation that should get updated.

https://github.com/PyCQA/pylint/blob/1f7f2e96c39ee89ee9fb18ab23f3dfbb9325c870/pylint/testutils/lint_module_test.py#L92-L96

Reply:
Thanks @cdce8p for reviewing even after we already merged, much appreciated!

I'll create a PR that updates the doc.

Reply:
@cdce8p Any good ideas about how to describe this. I have been trying for 5 minutes but can't come up with a simple but correct sentence 😆

Reply:
`` "max_pyver": Starting from that python version for the test won't be run`` ?

Reply:
See https://github.com/PyCQA/pylint/pull/5411